Sources/DispatchQueue.swift

```diff
@@ -9,17 +9,25 @@ import Foundation
 
 public struct DispatchQueue : DispatchObject, DispatchResumable {
     
-    public let queue: dispatch_queue_t!
-
-    public var rawValue: dispatch_object_t! {
-        return queue
+    public typealias RawValue = dispatch_queue_t
```

Comment:
This ```typealias``` is redundant, it gets inferred automatically when the ```DispatchObject``` protocol is adopted. Introducing it here declares an additional nested type ```DispatchQueue.RawValue``` which is undesirable.
Please, declare ```var rawValue: dispatch_queue_t``` and remove ```RawValue``` declaration at the struct level.